src/sonic-yang-models/yang-models/sonic-static-route.yang

```diff
@@ -95,6 +95,12 @@ module sonic-static-route {
           description
             "When interface is specified, forwarding happens through it";
         }
+        leaf advertise {
+          type string {

```

Comment:
Can we use boolean?

https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L259-L263